Source/Core/DolphinQt/Translation.cpp

```diff
@@ -264,6 +264,9 @@ static bool TryInstallTranslator(const QString& exact_language_code)
 #elif defined __APPLE__
         File::GetBundleDirectory() +
         StringFromFormat("/Contents/Resources/%s.lproj/dolphin-emu.mo", lang.c_str())
+#elif defined LINUX_LOCAL_DEV
+        File::GetExeDirectory() +
+        StringFromFormat("/../Source/Core/DolphinQt/%s/dolphin-emu.mo", lang.c_str())
```

Comment:
Why would the mo files be in the Source directory?

Reply:
Because that's how where our build system places them right now?

```shell
$ find -name *.mo
./Source/Core/DolphinQt/ar/dolphin-emu.mo
./Source/Core/DolphinQt/ca/dolphin-emu.mo
./Source/Core/DolphinQt/cs/dolphin-emu.mo
./Source/Core/DolphinQt/da/dolphin-emu.mo
...
```